content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,64 @@
 title: Kubernetes vs Docker
 linkTitle: Kubernetes vs Docker
 weight: 5
+aliases:
+  -  /hosting/vertical-vs-horizontal
 ---
 
 {{< hextra/hero-subtitle >}}
   Options for installing CHT applications
 {{< /hextra/hero-subtitle >}}
 
-Since the release of CHT Core 4.0.0 in late 2022, Medic has been perfecting the hosting for the toolkit to balance the need for high uptimes so CHWs can always deliver care while having an easy and approachable technical back end hosting solution.  While initially [Docker Compose](https://docs.docker.com/compose/) with an [overlay network](https://docs.docker.com/compose/networking/#multi-host-networking) was thought to be our goto solution, field testing this overlay networks in production has shown them to unreliable.  As such, on this site you will find documentation for both Docker Compose for smaller deployments (with no overlay network) and Kubernetes for larger deployments.
+Medic recommends [Docker](/hosting/4.x/docker/) for most deployments. Only use [Kubernetes](/hosting/4.x/kubernetes/) for large, multi-tenant or specific hardware constrained deployments.
```

Comment:
```suggestion
Medic recommends [Docker](/hosting/4.x/docker/) for small scale deployments. Use [Kubernetes](/hosting/4.x/kubernetes/) for large, multi-tenant or specific hardware constrained deployments.
```

Reply:
Your sentence assumes most of CHT deployments are small scale deployments which I am unsure is accurate.

Reply:
My sentence actually does not assume most CHT deployments are small scale (eg only 100s of CHWs).  We have proven that Docker can support large scale deployments.  We know that all CHT deployments that are this shape can be run Docker.  Removing "only" here I think softens it enough.

fixed in 65ee12f7